components/layout/display_list_builder.rs

```diff
@@ -1041,6 +1030,24 @@ fn calculate_inner_bounds(mut bounds: Rect<Au>, offsets: SideOffsets2D<Au>) -> R
     bounds
 }
 
+fn calculate_inner_border_radii(
+    mut radii: BorderRadii<Au>,
+    offsets: SideOffsets2D<Au>
+    ) -> BorderRadii<Au> {
```

Comment:
Minor nit:

I think the indentation @emilio means is

```rust
fn calculate_inner_border_radii(
    mut radii: BorderRadii<Au>,
    offsets: SideOffsets2D<Au>
) -> BorderRadii<Au> {
```

Reply:
Yeah, that's right, that's the indentation rustfmt would use for multi-line functions.